src/sqlancer/cockroachdb/CockroachDBSchema.java

```diff
@@ -24,7 +25,12 @@ public enum CockroachDBDataType {
         TIMETZ, ARRAY;
 
         public static CockroachDBDataType getRandom() {
-            return Randomly.fromOptions(values());
+            // return Randomly.fromOptions(values());
+            // INTERVAL and JSONB can trigger a number of known issues, e.g., [1], so we skip them.
+            // [1] https://github.com/cockroachdb/cockroach/issues/45993
```

Comment:
Would it be possible to adopt this scheme? https://github.com/sqlancer/sqlancer/blob/main/CONTRIBUTING.md#unfixed-bugs

Reply:
(also for some of the other cases)